### DISCUSSION TITLE
Add User Favorites to File Library

### DISCUSSION THREAD

**Discussion by benhaynes** [directus/directus#3120] [Feature Requests]
The only bummer here is that this will require a new DB table, but this gets us much closer to a DAM, and is well worth it.

The star toggle can either go in the header as an action button (File Detail), or as a normal interface (Icon Toggle) to the right of "Location".

I'll also add a small star to the thumbnails on the browse page for easier identification.

----

This could also be structured as a global M2X favorites table... so you can favorite anything in the system. Thanks @rijkvanzanten!

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
